templates/ts/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "%APPNAME%",
+  "version": "0.0.0",
+  "description": "",
+  "author": "",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc && bos-workspace build",
+    "test": "tsc --noEmit",
+    "tsc": "tsc --noEmit"
+  },
+  "dependencies": {
```

Comment:
This one doesn't work, it doesn't have bos-workspace installed on it

Let's remove the typescript template and just have two -- "single" and "multi" -- they can even be drawn from the `/examples` directory